gst/nnstreamer/tensor_common.h

```diff
@@ -195,5 +195,38 @@ gst_tensor_pad_possible_caps_from_config (GstPad * pad, const GstTensorsConfig *
 extern gboolean
 gst_tensor_pad_caps_is_flexible (GstPad * pad);
 
+/******************************************************
+ ************ Commonly used debugging macros **********
+ ******************************************************
+ */
+/**
+ * @brief Macro for debug message.
+ */
+#define silent_debug(self, ...) do { \
+    if (DBG) { \
+      GST_DEBUG_OBJECT (self, __VA_ARGS__); \
+    } \
+  } while (0)
+
+/**
+ * @brief Macro for capability debug message.
+ */
+#define silent_debug_caps(self, caps, msg) do { \
+  if (DBG) { \
+    if (caps) { \
+      GstStructure *caps_s; \
```

Comment:
suggestion : use gst_caps_to_string(caps) it simply returns caps string.

Reply:
Applied.